.travis.yml

```diff
@@ -2,24 +2,18 @@ language: haskell
 ghc:
   - '7.8'
 
-sudo: false
+sudo: true
```

Comment:
This will disable the caching entirely. I just implemented a stack-based cached build here:

https://github.com/fpco/wai-middleware-crowd/blob/master/.travis.yml

Any objection to just using that approach?